spyder/plugins/ipythonconsole/api.py

```diff
@@ -41,3 +41,89 @@ class IPythonConsolePyConfiguration(TypedDict):
     # If not None, then the console will use an alternative run method
     # (e.g. `runfile`, `debugfile` or `debugcell`).
     run_method: NotRequired[str]
+
+
+class IPythonConsoleWidgetActions:
```

Comment:
I moved all IPython console enums here because I need to use some of them to build the consoles context menu in ShellWidget.